ui/apps/platform/src/hooks/useURLPagination.ts

```diff
@@ -1,7 +1,7 @@
 import { useCallback } from 'react';
 import useURLParameter from './useURLParameter';
 
-type UseURLPaginationResult = {
+export type UseURLPaginationResult = {
```

Comment:
I'm finding that any time a function is exported we will eventually need the parameter and return types to be exported as well...